packages/effect/dtslint/Record.tst.ts

```diff
@@ -589,4 +589,20 @@ describe("Record", () => {
     expect(Record.intersection(string$structAB, { b: 2 }, (a, _) => a))
       .type.toBe<Record<"b", number>>()
   })
+
+  it("findFirstEntry", () => {
+    expect(Record.findFirstEntry(string$numbersOrStrings, (a, _) => predicateNumbersOrStrings(a)))
+      .type.toBe<Option.Option<[string, string | number]>>()
+    expect(pipe(string$numbersOrStrings, Record.findFirstEntry((a, _) => predicateNumbersOrStrings(a))))
+      .type.toBe<Option.Option<[string, string | number]>>()
+    expect(Record.findFirstEntry(string$numbersOrStrings, (a, _) => Predicate.isString(a)))
```

Comment:
This expect fails with Typescript 5.4.5. If you have any tip on how to fix it, I'm all ears!

Reply:
it seems like we don't test on all typescript versions anymore